crypto/keys/internal/ecdsa/privkey.go

```diff
@@ -38,10 +87,25 @@ func (sk *PrivKey) Bytes() []byte {
 	return bz
 }
 
-// Sign hashes and signs the message usign ECDSA. Implements SDK PrivKey interface.
+// Sign hashes and signs the message using ECDSA. Implements SDK
+// PrivKey interface.
+// NOTE: this now calls the ecdsa Sign function
+// (not method!) directly as the s value of the signature is needed to
+// low-s normalize the signature value
```

Comment:
let's add a link to the source (issue) explaining why we need to normalize.

Reply:
Added.